Mathlib/LinearAlgebra/Matrix/PosDef.lean

```diff
@@ -241,6 +253,14 @@ lemma posSemidef_iff_eq_transpose_mul_self [DecidableEq n] {A : Matrix n n 𝕜}
   simp_rw [← PosSemidef.sq_sqrt hA, pow_two]
   rw [hA.posSemidef_sqrt.1]
 
+lemma posSemidef_iff_eigenvalues_nonneg [DecidableEq n] {A : Matrix n n 𝕜} :
+    PosSemidef A ↔ ∃ (h : IsHermitian A), ∀ i : n, 0 ≤ h.eigenvalues i := by
```

Comment:
Thanks for adding this. Since we already have the `→` direction, maybe you could just add the `←` direction like so, in order to get dot notation:
```lean
lemma posSemidef_of_eigenvalues_nonneg [DecidableEq n] {A : Matrix n n 𝕜}
    (hA : A.IsHermitian) (h : ∀ i : n, 0 ≤ hA.eigenvalues i) : PosSemidef A := sorry
```